x-pack/plugins/serverless_search/public/application/components/languages/java.ts

```diff
@@ -14,14 +14,14 @@ export const javaDefinition: LanguageDefinition = {
   iconType: 'java.svg',
   github: {
     label: i18n.translate('xpack.serverlessSearch.languages.java.githubLabel', {
-      defaultMessage: 'elasticsearch-java-serverless',
+      defaultMessage: 'elasticsearch-java',
     }),
-    link: 'https://github.com/elastic/elasticsearch-java/tree/main/java-client-serverless',
+    link: 'https://github.com/elastic/elasticsearch-java/tree/main/java-client',
   },
   // Code Snippets,
   installClient: `dependencies {
-    implementation 'co.elastic.clients:elasticsearch-java-serverless:1.0.0-20231031'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.3'
+    implementation 'co.elastic.clients:elasticsearch-java:${elasticsearchVersion}'
```

Comment:
is there a "latest" tag that we can use here? elasticsearchVersion variable doesn't exist